source/exe/BUILD

```diff
@@ -150,25 +149,21 @@ envoy_cc_library(
 
 envoy_cc_library(
     name = "envoy_main_common_with_core_extensions_lib",
-    srcs = [
-        "main_common.cc",
-    ],
-    hdrs = [
-        "main_common.h",
-    ],
+    srcs = ["main_common.cc"],
+    hdrs = ["main_common.h"],
     deps = [
-        "//source/server/config_validation:server_lib",
```

Comment:
cc @phlax I think this version should not have passed format checks.  Did we lose buildifier on our format checks?

Reply:
i believe it works the same as it always did

afaict the queue was cancelled before the check was run

it the `./tools/code_format/check_format.py` script that runs it i think